net/phantap/Makefile

```diff
@@ -0,0 +1,87 @@
+# Copyright (C) 2019 Diana Dragusin <diana.dragusin@nccgroup.com>
+# Copyright (C) 2019 Etienne Champetier <champetier.etienne@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v3.
+# See <http://www.gnu.org/licenses/> for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=phantap
+PKG_VERSION:=2019.07.30
```

Comment:
Probably should use PKG_SOURCE_DATE here. That way PKG_SOURCE can be removed.

Reply:
Thanks for the tips, using PKG_SOURCE_DATE & removed PKG_SOURCE